### DISCUSSION TITLE
Possibilty for translating toots in-app

### DISCUSSION THREAD

**Discussion by M-Stenzel** [mastodon/mastodon#17119] [General]
Hi team,

it would be so fancy to have the chance to translate toots w/in the web app or the android app.
Google translate is nothing to go for, but what about here?

https://www.deepl.com/pro-api?cta=header-pro-api/

Martin.

**Comment by Gargron** [comment]
We could add it in [mastodon/mastodon-ios ](https://github.com/mastodon/mastodon-ios) more easily than in the web app, because then we'd take care of the API keys and paying for API access; in the web app, admins would be individually liable for setting up DeepL integrations and it might be a bit chaotic.
